code/modules/antagonists/changeling/powers/adrenaline.dm

```diff
@@ -9,6 +9,7 @@
 
 //Recover from stuns.
 /obj/effect/proc_holder/changeling/adrenaline/sting_action(mob/living/user)
+	..()
```

Comment:
Even if there isn't a return value yet we should still be using `. = ..()` for ease of refactoring later. It's harmless future proofing.